component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java

```diff
@@ -454,6 +454,11 @@ public Response getUsersOrConnectionsByAdvancedFilter(@Context UriInfo uriInfo,
           filter.setUserType(userType);
         }
       }
+      if (settings != null) {
+        List<Identity> excludedIdentityList = new ArrayList();
+        excludedIdentityList.add(target);
+        filter.setExcludedIdentityList(excludedIdentityList);
```

Comment:
Could this be improved like this, WDYT?
```suggestion
        filter.setExcludedIdentityList(Collections.singletonList(target));
```